pkg/db/remoteRequest.go

```diff
@@ -72,7 +72,7 @@ func createRequest(dbKeys []string, values [][]byte, signer RequestSigner) (*req
 	}
 	if sig == nil {
 		log.Error("Signature was not generated")
```

Comment:
```suggestion
```
not related to the PR , but we can remove this log entry, no point on having an error and a log, it is kind of duplicate.